src/components/Answers.tsx

```diff
@@ -14,6 +15,7 @@ const Answers = ({
   showCorrect: boolean;
   handleSelection: (answer: AnswerType) => void;
 }) => {
+  const { t } = useTranslation();
```

Comment:
Could we wrap this in a util function so we don't have to write this line in every component? Instead, we import it from one common place. It would also make future refactoring easier, because it is in one place.

Reply:
Not sure how to do this.. can you sketch out what you have in mind? 

Having this makes more sense if we split the translations into multiple files (https://react.i18next.com/guides/multiple-translation-files); one file for each component. Then, this would be 
 const { t } = useTranslation(file_name); But as we don't have that many translations I decided to go with one unique file.